has_position_limits: false
min_position: 0.0
max_position: 0.0

has_velocity_limits: false
max_velocity: 0.0

has_acceleration_limits: false
max_acceleration: 0.0

has_jerk_limits: false
max_jerk: 0.0

has_effort_limits: false
max_effort: 0.0